operating_unit/security/operating_unit_security.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data noupdate="1">
+
+    <record id="group_multi_operating_unit" model="res.groups">
+        <field name="name">Multi Operating Unit</field>
+        <field name="category_id" ref="base.module_category_usability"/>
+    </record>
+
+    <record id="operating_unit_comp_rule" model="ir.rule">
+        <field name="name">Operating Unit Entry</field>
+        <field name="model_id" ref="model_operating_unit"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+
```

Comment:
This would mean that users belong to those OU. 

Putting a restriction to the ability to see other users could but a barrier to assigning them to OU independent models. For example, I see Projects as a cross-OU function. I'm thinking that a project manager may want to assign tasks to users that can potentially belong to other OU's. Or users on a project might want to work together, even when they have rights to operate on separate OU's.

So OU's can partition transactional data. But partitioning master data (e.g. customers, products, users,...) may generate a too restrictive environment.